boards/arm/mec1501modular_assy6885/pinmux.c

```diff
@@ -382,17 +382,17 @@ static int board_pinmux_init(struct device *dev)
 #if DT_HAS_NODE(DT_INST(0, microchip_xec_qmspi))
 	mchp_pcr_periph_slp_ctrl(PCR_QMSPI, MCHP_PCR_SLEEP_DIS);
 
-#if DT_SPI_XEC_QMSPI_0_PORT_SEL == 0
+#if DT_INST_PROP(0, port_sel) == 0
 	/* Port 0: Shared SPI pins. Shared has two chip selects */
-#if DT_SPI_XEC_QMSPI_0_CHIP_SELECT == 0
+#if DT_INST_PROP(0, chip_select) == 0
```

Comment:
These should be something like `DT_PROP(DT_INST(0,microchip_xec_qmspi), port_sel)`